### ISSUE TITLE
Message: unknown error: window.wave_viewIcons is not a function

### ISSUE THREAD

**Issue by emanuelsimon-qa** [datakurre/robotframework-wavelibrary#1]
on `WAVELibrary/keywords.robot` keywords using  ```    Execute Javascript
    ...    return (function(){ window.wave_view(); return true; })();``` functions seem to be broken.

Not sure if relevant but WebAIM has updated their toolbar to https://addons.mozilla.org/en-US/firefox/addon/wave-accessibility-tool/.

**Comment by datakurre** [comment]
@emanuelstorres Yes, the add-on has updated. This library was written with years old version, before Firefox supported "web extensions".

I was unaware that there is now modern version of the extension for Firefox. Thanks for the heads up.

Unfortunately, personally I have not had (and may not have for some time) time to check if this approach could still work with "web extensions" and the new wave add-on.

**Comment by zrqsar** [comment]
Not sure if there was any update released, but i tried using this lib and facing same error, any pointers here please

Error : 

Executing JavaScript: return (function(){ window.wave_viewIcons(); return true; })(); Without any arguments. |  

JavascriptException: Message: javascript error: window.wave_viewIcons is not a function
  (Session info: chrome=91.0.4472.124)

**Comment by datakurre** [comment]
@zrqsar No updates, unfortunately. This would only work with pre-quantum Firefox, and I am not sure if Internet still works with that.

[I have automated](https://rpa.pages.kopla.jyu.fi/wavebot/) recent Wave add-on on Firefox and Chromium, but have not managed to publish that.  I may not be able to fix this package, because 1) we don't have permission to distribute later Wave add-ons, 2) Wave add-on cannot be executed from Seleniun due to browser security, but it must be executed with some desktop automation (to really click the browser toolbar icon for Wave add-on).

**Comment by Karthi-Murugesan** [comment]
Hi @zrqsar @emanuelsimon-qa  where you able to fix this issue or there is alternatives found..?

**Comment by datakurre** [comment]
I have no updates since my last comment.

There was an alternative presented at Robocon for the browser library https://github.com/MarketSquare/robotframework-browser-extensions/tree/main/Axe-Core
